src/assets/stylesheets/ui-root.scss

```diff
@@ -228,34 +228,58 @@ body.vr-mode {
   font-style: italic;
 }
 
-:local(.message-entry-submit) {
-  @extend %action-button;
+:local(.message-entry-button) {
+  color: $action-color;
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  outline-style: none;
+  background-color: white;
   position: absolute;
-  right: 10px;
-  height: 32px;
-  min-width: 80px;
-  bottom: 8px;
-  border-radius: 10px;
+  min-width: 40px;
+  height: 100%;
+  bottom: 0px;
+  border: 0px;
+  font-size: 1.5em;
+  cursor: pointer;
 }
 
-:local(.message-entry-in-room) {
-  @media(max-width: 900px) {
-    display:none;
-  }
+:local(.message-entry-button-in-room) {
+  color: $dark-grey;
+}
 
+:local(.message-entry-button-in-room):hover {
+  color: $action-color;
+}
+
+:local(.message-entry-submit) {
+  right: 15px;
+}
+
+:local(.message-entry-upload) {
+  margin-bottom: 1px;
+  right: 55px;
```

Comment:
Would also suggest using regular flex here if we can, instead of rearranging elements with `right`.